src/Controller/CallbackApiController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace GoApptiv\FileManagement\Controller;
+
+use Illuminate\Http\Request;
+use GoApptiv\FileManagement\Services\FileManagement\FileManagementService;
+use Illuminate\Routing\Controller;
+use Illuminate\Support\Facades\Log;
+
+class CallbackApiController extends Controller
```

Comment:
It can be FileVariantController instead of CallbackApiController

Reply:
Done.